client/java-armeria/src/main/java/com/linecorp/centraldogma/client/armeria/ArmeriaCentralDogma.java

```diff
@@ -978,7 +979,9 @@ private static ArrayNode toJson(Iterable<? extends Change<?>> changes) {
             changeNode.put("path", c.path());
             changeNode.put("type", c.type().name());
             final Class<?> contentType = c.type().contentType();
-            if (contentType == JsonNode.class) {
+            if (isJson5(c.path())) {
```

Comment:
How about adding the `FileType` mentioned to `Change` so that we check file type more statically?
```java
public interface Change<T> {

    FileType fileType() {
         ...
    }
}
```